stock_valuation_account_manual_adjustment/README.rst

```diff
@@ -0,0 +1,84 @@
+.. image:: https://img.shields.io/badge/license-AGPLv3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl.html
+   :alt: License: AGPL-3
+
+=========================================
+Stock Inventory Account Manual Adjustment
+=========================================
+
+This module shows in the product inventory stock value and the accounting
+value, and allows to create accounting adjustment entries to align the
+accounting value to match with the inventory stock value.
+
+Configuration
+=============
+
+Assign the group 'Stock Valuation Account Manual Adjustments' to the users
+that should be allowed to list the valuation discrepancies and to reconcile
+them.
+
+Usage
+=====
+
+* Go to the list of products or product variants. You can see in the tree
+  view the inventory value and the accounting value.
```

Comment:
Inventory Value, Accounting Value, and Discrepancy can only be seen on *Real Time Inventory Valuation*